roles/debian/pam_ldap/templates/ldap.conf.j2

```diff
@@ -1,9 +1,11 @@
 BASE {{ pam_ldap.lookup_base }}
-URI	{{ pam_ldap.endpoints | join(' ') }}
-{% if pam_ldap.ssl_certificate is defined and pam_ldap.ssl_certificate %}
+URI {{ pam_ldap.endpoints | join(' ') }}
+{% if pam_ldap.ssl_use_system_ca %}
 TLS_CACERT /etc/ssl/certs/ca-certificates.crt
+{% elif pam_ldap.ssl_certificate %}
```

Comment:
Sorry, not your bug, it's always been there, but this needs to be `{% elif pam_ldap.ssl_certificate | length > 0 %}` or it will always resolve to true.